src/action.c

```diff
@@ -321,6 +321,10 @@ action_arg_from_xml_node(struct action *action, const char *nodename, const char
 			action_arg_add_str(action, argument, content);
 			goto cleanup;
 		}
+		if (!strcmp(argument, "atCursor")) {
+			action_arg_add_bool(action, argument, parse_bool(content, true));
```

Comment:
I think this has to be `strcasecmp()`, otherwise this will never match (as argument is lower-cased automatically afair)

Reply:
You're right; this baffled me in testing for a few minutes. Fixed.